.ci/run_nccl_test.sh

```diff
@@ -0,0 +1,400 @@
+#!/bin/bash -l
+
+set -o pipefail
+
+if [ -n "$DEBUG" ]
+then
+    set -x
+fi
+
+HOSTNAME=`hostname -s`
+echo "DEBUG: HOSTNAME = $HOSTNAME"
+
+module load dev/cuda10.0
```

Comment:
`hpcx-gcc` (HPCX v2.5) uses CUDA10.1.

Reply:
Right now I'm considering mainly CUDA 10.0 because as far as I know there might be some issues with Mellanox ML solutions and CUDA 10.1 (need to double check).